### DISCUSSION TITLE
Rotate screen if user click on full screen

### DISCUSSION THREAD

**Discussion by gotipath** [yoriiis/vlitejs#110] [Ideas]
On mobile, if you click to full screen on any video of _m.youtube.com_ it will automatically rotate and follow the gyroscope sensor direction. Even if your auto-rotation is disabled from the setting. This feature gives a good viewing experience to the user.

**Comment by yoriiis** [comment]
Thanks @gotipath for the idea, I will work on this!

**Reply by yoriiis** [reply]
The feature is now available on mobile for Chrome and Edge, for Firefox this is native behavior. You can get a preview on the Vercel deployment here: https://vlitejs.vercel.app/html5

It will be deployed on the next version!
